### PR TITLE
NEO-1365 fix keyboard arrow up and down in single select, searchable or not

### PR DESCRIPTION
**Pull request by yangAtSpoken**
[Single Select](https://deploy-preview-106--neo-react-library-storybook.netlify.app/?path=/story/components-select--basic-selects)

[Single Searchable Select](https://deploy-preview-106--neo-react-library-storybook.netlify.app/?path=/story/components-select--searchable)

**Before tagging the team for a review, I have done the following:**
- [x] reviewed my code changes
- [x] ensured that all tests pass
- [x] updated the link at the top of this PR (or remove it if not applicable)
- [x] added any important information to this PR (description, images, GIFs, ect.)
- [x] tagged Matt if any visual changes have occurred
- [x] done an accessibility check on my work

Both the Single Select and Searchable Single Select are not scrolling with up and down arrows.  This PR fixes it by moving menu props to parent div of the ul element.  Also role="group" has to be added to the ul element to keep a11y checker happy.

### CODE REVIEW

**Review by mrazian85 — APPROVED** [review]
Looks like it is working great now 👍

File: src/components/Select/InternalComponents/SingleSelect.tsx [hunk 1]
```diff
@@ -38,8 +38,12 @@ export const SingleSelect = () => {
...
       </button>
 
-      <div className="neo-multiselect__content">
-        <ul aria-label={ariaLabel} {...getMenuProps()}>
+      <div
+        className="neo-multiselect__content"
+        aria-label={ariaLabel}
+        {...getMenuProps()}
+      >
+        <ul role="group">
```
Comment (joe-s-avaya) [inline src/components/Select/InternalComponents/SingleSelect.tsx:46]: surprised that this was needed, great find!

File: src/components/Select/Select.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    cy.contains("Select One").click(); // click the first select one
+
+    cy.get("[role='listbox']")
+      .first()
+      .within(() => {
+        cy.get("ul li").last().should("not.be.visible");
+        cy.get("ul li")
+          .first()
+          .type(
+            "{downArrow}{downArrow}{downArrow}{downArrow}{downArrow}{downArrow}{downArrow}"
```
[the remark this answers is not in the document]
Comment (joe-s-avaya) [reply src/components/Select/Select.cy.tsx:16]: second this 👍 
great tests

**Review by joe-s-avaya — APPROVED** [review]
nice

**Comment by matthewAvaya** [comment]
- Remove all placeholder text due to cognitive accessibility issues.
- Spacing issues on all multiple select versions. When there are multiple chips selected, the top padding should match the version that only has a single chip.
- On multiple select, the cursor should be positioned next to the last chip rather than on a new separate line. 
- On multiple select versions, the hover states are cutting off the focus blue border (left and right sides). 

Accessibility issues:
- Basic Selects: After selecting an item, the screen reader is reading some of the objects twice... 'object object has been selected, object object has been selected'.
- When I tab to a multiple select that already has a chip, the focus should go to the entire input field and then on next tab go directly to the close icon on the first chip. See example from [Carbon Design](https://carbondesignsystem.com/components/dropdown/code/#main-content).

**Comment by yangAtSpoken** [comment]
> * Remove all placeholder text due to cognitive accessibility issues.

@matthewAvaya Turns out it is not trivial to do this.  I have created https://jira.forge.avaya.com/browse/NEO-1387 to track this.

**Comment by yangAtSpoken** [comment]
> * Spacing issues on all multiple select versions. When there are multiple chips selected, the top padding should match the version that only has a single chip.
> * On multiple select, the cursor should be positioned next to the last chip rather than on a new separate line.
> * On multiple select versions, the hover states are cutting off the focus blue border (left and right sides).

These are tracked in https://jira.forge.avaya.com/browse/NEO-1388

**Comment by yangAtSpoken** [comment]
> * Basic Selects: After selecting an item, the screen reader is reading some of the objects twice... 'object object has been selected, object object has been selected'.

Traced by https://jira.forge.avaya.com/browse/NEO-1389

**Comment by yangAtSpoken** [comment]
Got @matthewAvaya 's permission to merge.

**Pull request merged**